src/indexProgram/types.ts

```diff
@@ -36,11 +20,31 @@ export type EsDonorDocument = {
   sangerVcsRunning: number;
   sangerVcsFailed: number;
 
-  processingStatus: DonorMolecularDataProcessingStatus;
-
   totalFilesCount: number;
   filesToQcCount: number;
 
+  releaseStatus: DonorMolecularDataReleaseStatus;
+  processingStatus: DonorMolecularDataProcessingStatus;
+};
+
+export type MongoDonorInfo = {
```

Comment:
maybe name this `ClinicalDonorInfo` sound? Mongo is kind of an implementation detail here